### PR TITLE
Added logic to load annoto completion conditions for annoto assighnments

### PR DESCRIPTION
**Pull request by IlyaRazuvaev**
https://app.clickup.com/t/86by58p2z

### CODE REVIEW

File: lib.php [hunk 1]
```diff
@@ -674,10 +675,34 @@ function local_annoto_coursemodule_standard_elements($formwrapper, $mform)
...
+        $typeid = $mform->getElementValue('typeid');
+        $lti_config = lti_get_type_config($typeid);
+        $tooldomain = $lti_config['tooldomain'];
+        $deploymentdomain = $settings->deploymentdomain;
+        if ($deploymentdomain === 'custom') {
+            $deploymentdomain = $settings->customdomain;
+        }
+
+        if (empty($deploymentdomain) || !isset($deploymentdomain)) {
+            $deploymentdomain = 'annoto';
```
Comment (genadis) [inline lib.php:699]: change to valid domain annoto.net

File: lib.php [hunk 2]
```diff
@@ -674,10 +675,34 @@ function local_annoto_coursemodule_standard_elements($formwrapper, $mform)
...
 
+    /*
+     *  tooldomain - parsed from lti_toolurl of the plugin that launched page
+     *  deploymentdomain, customdomain - domain recieved from the local 
+     *  annoto plagin integration
+     *  domain
+     */
+    if ($modulename === 'lti') {
+        $typeid = $mform->getElementValue('typeid');
+        $lti_config = lti_get_type_config($typeid);
```
Comment (genadis) [inline lib.php:691]: fix variable name (run linter)

**Review by genadis — CHANGES_REQUESTED** [review]
why the source is not master branch but instead 5.1.0 tag?

**Comment by IlyaRazuvaev** [comment]
Can see only one branch now. Shouldn't be a problem as I see based on the file changes
image

**Review by genadis — APPROVED** [review]

**Pull request merged**